tree/dataframe/inc/ROOT/RDF/RLoopManager.hxx

```diff
@@ -11,11 +11,15 @@
 #ifndef ROOT_RLOOPMANAGER
 #define ROOT_RLOOPMANAGER
 
+#include "ROOT/InternalTreeUtils.hxx" // RNoCleanupNotifier
```

Comment:
Where is this header used ?

Reply:
Good catch, indeed `#include "ROOT/RDF/RColumnReaderBase.hxx"` is not needed as well. Thank you!